sdk/tables/azure-data-tables/test/ut/enum_operators_test.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "enum_operators_test.hpp"
+
+using namespace Azure::Data::Tables::Sas;
+using namespace Azure::Data::Tables;
+namespace Azure { namespace Data { namespace Test {
+  TEST(EnumOperator, AndTest)
+  {
+    {
+      auto val = TestEnum::One & TestEnum::Two;
```

Comment:
When you add constexpr before each operator, you can also add at least one per each test for an operator, so that should someone drop `constexpr`, the test would fail to compile.
```suggestion
      constexpr auto val = TestEnum::One & TestEnum::Two;
```